src/chmfontdialog.cpp

```diff
@@ -111,8 +115,17 @@ void CHMFontDialog::UpdatePreview()
     _test->SetPage(test_page);
 }
 
-void CHMFontDialog::OnUpdate(wxFontPickerEvent&)
+void CHMFontDialog::OnUpdate(wxFontPickerEvent& event)
 {
+    if (event.GetId() == ID_FixedFont) {
+        wxFont selectedFont = _normalFControl->GetSelectedFont();
+        selectedFont.SetPointSize(event.GetFont().GetPointSize());
+        _normalFControl->SetSelectedFont(selectedFont);
+    } else if (event.GetId() == ID_NormalFont) {
+        wxFont selectedFont = _fixedFControl->GetSelectedFont();
+        selectedFont.SetPointSize(event.GetFont().GetPointSize());
+        _fixedFControl->SetSelectedFont(selectedFont);
+    }
```

Comment:
But I think we don't need the IDs at all, and the more vebose `if ()` code can simply be replaced by:
```c++
_normalFont.SetPointSize(event.GetFont().GetPointSize());
_fixedFont.SetPointSize(event.GetFont().GetPointSize());
```
as in:
```diff
diff --git a/src/chmfontdialog.cpp b/src/chmfontdialog.cpp
index 5e57efc..ee4fe80 100644
--- a/src/chmfontdialog.cpp
+++ b/src/chmfontdialog.cpp
@@ -117,6 +117,7 @@ void CHMFontDialog::UpdatePreview()

 void CHMFontDialog::OnUpdate(wxFontPickerEvent& event)
 {
+    /*
     if (event.GetId() == ID_FixedFont) {
         wxFont selectedFont = _normalFControl->GetSelectedFont();
         selectedFont.SetPointSize(event.GetFont().GetPointSize());
@@ -126,9 +127,14 @@ void CHMFontDialog::OnUpdate(wxFontPickerEvent& event)
         selectedFont.SetPointSize(event.GetFont().GetPointSize());
         _fixedFControl->SetSelectedFont(selectedFont);
     }
+    */
+
     _normalFont = _normalFControl->GetSelectedFont();
     _fixedFont  = _fixedFControl->GetSelectedFont();

+    _normalFont.SetPointSize(event.GetFont().GetPointSize());
+    _fixedFont.SetPointSize(event.GetFont().GetPointSize());
+
     UpdatePreview();
 }
```
Am I missing something?

Reply:
That works too. I changed it.